AutomationPractice/features/Pages/CreateAccountPage.py

```diff
@@ -5,13 +5,13 @@
 from selenium.webdriver.support import expected_conditions as EC
 from ultilities.Random import Random
 import Locator
+from features.browsers.browser import Browser
 
 
-class CreateAccount():
+class CreateAccount(Browser):
     rd = Random()
     def __init__(self, driver):
         self.driver = driver
-
     ### Create Element
     def getGenderMaleBtn(self):
         return self.driver.find_element(Locator.gender_male[0], Locator.gender_male[1])
```

Comment:
we can add a locator variable to shorten the locator argument input like :

def get_gender_male_btn(self):
         locator = Locator.gender_male
         return self.driver.find_element(locator[0], locator[1])